IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo

```diff
@@ -1470,6 +1470,17 @@ For instance, the unit must be
 </ul>
 </li>
 <li>
+<p>
+Hourly and subhourly timestamp are handled in a different way in <code>.epw</code> files.
+From the EnergyPlus Auxiliary Programs Document (v9.3.0, p. 63):
+In hourly data the minute field can be <code>00</code> or <code>60</code>. In this case as mentioned in the previous section, the weather data
+is reported at the hourly value and the minute field has to be ignored, wrinting <code>1,60</code> or <code>1,00</code> is equivalent.
```

Comment:
And sure about `1,60`? It really looks uncommon. Edit: I just saw the next section, so `1,60` might be equivalent to `02:00`?

Reply:
@mlauster : Yes, we are sure and agree that it look inconsistent. @EttoreZ double-checked with the EnergyPlus support and inquired specifically about this inconsistency. This is how EnergyPlus handles the minute field.